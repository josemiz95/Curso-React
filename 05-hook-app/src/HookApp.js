import React from 'react';
// import { CounterApp } from './components/01-useState/CounterApp';
// import { CounterWithCustomHooks } from './components/01-useState/CounterWithCustomHooks';
// import { SimpleForm } from './components/02-useEffect/SimpleForm';
// import { FormWithCustomHook } from './components/02-useEffect/FormWithCustomHook';
// import { MultipleCustomHooks } from './components/03-examples/MultipleCustomHooks';
// import { FocusScreen } from './components/04-useRef/FocusScreen';
// import { RealExampleRef } from './components/04-useRef/RealExampleRef';
// import { LayoutEffect } from './components/05-useLayoutEffect/LayoutEffect';
// import { Memorize } from './components/06-memos/Memorize';
import { MemoHook } from './components/06-memos/MemoHook';


const HookApp = () => {
  return (
    <MemoHook />
  )
}

export {
    HookApp as default
}