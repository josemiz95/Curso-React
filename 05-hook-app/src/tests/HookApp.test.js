import { shallow } from "enzyme";
import HookApp from "../HookApp";


describe('Test <HookApp/>', () => {
    test('Should coincide with snapshoot', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })
});