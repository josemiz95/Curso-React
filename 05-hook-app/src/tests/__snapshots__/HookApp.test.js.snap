// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <HookApp/> Should coincide with snapshoot 1`] = `<MainApp />`;
